artcommon/artcommonlib/metadata.py

```diff
@@ -481,8 +481,8 @@ async def get_latest_konflux_build(self,
             return default
         return build_record
 
-    async def get_latest_brew_build_async(self, **kwargs) -> Model:
-        return Model(await asyncio.to_thread(self.get_latest_brew_build, **kwargs))
```

Comment:
get_latest_brew_build has a default parameter used to define the return value while the model breaks the type